README.md

```diff
@@ -1,2 +1,2 @@
 # project_G
-Game on Python!
+Dating SIM using Python
```

Comment:
add a bit more from the issue and how to run it (`python -m sim.main_game`)

haha cool

<img width="1312" alt="image" src="https://github.com/abaddonpuff/project_G/assets/387927/cc5408dc-665a-47e2-98a0-50311c4a25a8">